source/future.tex

```diff
@@ -1468,8 +1468,7 @@
 only if the type \tcode{T} is any of the following:
 \begin{itemize}
 \item a function type or a pointer to function type taking one argument of type \tcode{T1}
-% FIXME: Should this be R T0::f() ?
-\item a pointer to member function \tcode{R T0::f} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
+\item a pointer to member function \tcode{R T0::f()} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
```

Comment:
i.e. Eelis's suggestion is not a normative change, the intention, and I believe what all implementations do, is to only define `argument_type` when it's a pointer to member function for a member function taking no parameters.

For a member function taking exactly one parameter `first_argument_type` and `second_argument_type` are defined.

For member functions taking two or more parameters none of those typedefs is defined.